src/page/analyze/__test__/AnalyzeDocumentPage.test.tsx

```
@@ -1,101 +1,54 @@
+vi.mock('@/components/Sidebar', () => ({
+  default: ({ isSidebarVisible }: { isSidebarVisible: boolean }) => (
+    <div data-testid="sidebar" data-visible={isSidebarVisible}>Sidebar</div>
+  )
+}));
 
-// 1) Mock Sidebar
-vi.mock("@/components/Sidebar", () => ({
-  __esModule: true,
-  default: (props: any) => (
+vi.mock('@/components/Dropzone', () => ({
+  default: ({ setPdfUrl }: { setPdfUrl: Function }) => (
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)